libnavigation-base/src/main/java/com/mapbox/navigation/base/route/Router.kt

```diff
@@ -1,9 +1,22 @@
 package com.mapbox.navigation.base.route
 
 import com.mapbox.geojson.Point
+import com.mapbox.navigation.base.route.model.Route
 
 interface Router {
 
-    fun getRoute(origin: Point, waypoints: List<Point>, callback: (route: Route) -> Unit)
+    fun getRoute(
+        origin: Point,
+        waypoints: List<Point>?,
```

Comment:
Same as in https://github.com/mapbox/mapbox-navigation-android/pull/2240#discussion_r344938183